cmd/agent/app/integrations.go

```diff
@@ -153,14 +144,6 @@ var removeCmd = &cobra.Command{
 	RunE:  remove,
 }
 
-var searchCmd = &cobra.Command{
```

Comment:
they weren't able before anyway, the repo didn't support searching, so using the command just errored. Since the command was hidden anyway, we can safely remove it.